tests/main/interfaces-framebuffer/task.yaml

```diff
@@ -11,20 +11,16 @@ prepare: |
     snap try $TESTSLIB/snaps/test-snapd-framebuffer
 
 execute: |
-    CONNECTED_PATTERN=":framebuffer +test-snapd-framebuffer"
-    DISCONNECTED_PATTERN="\- +test-snapd-framebuffer:framebuffer"
+    echo "The plug is not connected by default"
+    snap interfaces | MATCH "\- +test-snapd-framebuffer:framebuffer"
```

Comment:
Seeing this over and over I start to agree with @plars that using `snap interfaces -I framebuffer` here and everywhere else would just make this less error prone.

Reply:
@zyga, I agree it is more robust, I'll implemet that in a new PR due to is it a big change.

Thanks for reviewing

Reply:
Makes sense to do it an follow-up. This one is already big.